app/views/CreateDiscussionView/index.tsx

```diff
@@ -149,35 +138,39 @@ class CreateChannelView extends React.Component<ICreateChannelViewProps, ICreate
 				contentContainerStyle={styles.container}
 				keyboardVerticalOffset={128}>
 				<StatusBar />
-				<SafeAreaView testID='create-discussion-view' style={styles.container}>
+				<SafeAreaView testID='create-discussion-view'>
 					<ScrollView {...scrollPersistTaps}>
-						<Text style={[styles.description, { color: themes[theme].fontSecondaryInfo }]}>{I18n.t('Discussion_Desc')}</Text>
-						<SelectChannel
-							server={server}
-							userId={user.id}
-							token={user.token}
-							initial={this.channel && { text: getRoomTitle(this.channel) }}
-							onChannelSelect={this.selectChannel}
-							blockUnauthenticatedAccess={blockUnauthenticatedAccess}
-							serverVersion={serverVersion}
-						/>
-						<FormTextInput
-							label={I18n.t('Discussion_name')}
-							testID='multi-select-discussion-name'
-							placeholder={I18n.t('A_meaningful_name_for_the_discussion_room')}
-							containerStyle={styles.inputStyle}
-							defaultValue={name}
-							onChangeText={(text: string) => this.setState({ name: text })}
-						/>
-						<SelectUsers
-							server={server}
-							userId={user.id}
-							token={user.token}
-							selected={users}
-							onUserSelect={this.selectUsers}
-							blockUnauthenticatedAccess={blockUnauthenticatedAccess}
-							serverVersion={serverVersion}
-						/>
+						<Text style={[styles.description, { color: themes[theme].fontDefault }]}>{I18n.t('Discussion_Desc')}</Text>
+						<View style={{ gap: 12, paddingTop: 12 }}>
+							<SelectChannel
+								required
```

Comment:
`SelectChannel` seems to be used only here, so no reason to overcomplicate with a prop that has always the same value.
I say we make it always required.

Reply:
adjusted.